### PR TITLE
feat: Support emitting JSON to topics

### PR DESCRIPTION
**Pull request by johanandren**
References #245

### CODE REVIEW

**Comment by johanandren** [comment]
Test are perhaps somewhat thin but in addition I have manually verified hitting `Produce` in snippets, which publishes JSON to a topic that the `Consume` consumes and logs 👍

File: sdk/test/action-handler-test.js [hunk 1]
```diff
@@ -130,7 +145,7 @@ describe('ActionHandler', () => {
   });
 
   // synchronous handlers
-
+  /*
```
Comment (pvlugter) [inline sdk/test/action-handler-test.js:148]: Assuming all the other tests were commented out for faster testing, but should still be enabled.

```suggestion
```

File: sdk/test/action-handler-test.js [hunk 2]
```diff
@@ -562,4 +577,25 @@ describe('ActionHandler', () => {
         'nested:promised:async:async:something',
       );
   });
+*/
```
Comment (pvlugter) [inline sdk/test/action-handler-test.js:580]:
```suggestion
```

**Review by pvlugter — APPROVED** [review]
LGTM!

File: sdk/test/action-handler-test.js [hunk 3]
```diff
@@ -130,7 +145,6 @@ describe('ActionHandler', () => {
   });
 
   // synchronous handlers
```
Comment (pvlugter) [inline sdk/test/action-handler-test.js:147]: Hmm, removed spacing as well, try adding it back.

```suggestion
  // synchronous handlers

```

File: sdk/test/action-handler-test.js [hunk 4]
```diff
@@ -562,4 +576,24 @@ describe('ActionHandler', () => {
         'nested:promised:async:async:something',
       );
   });
+  it('should reply with Akkaserverless JSON for unary methods returning Any', () => {
```
Comment (pvlugter) [inline sdk/test/action-handler-test.js:579]:
```suggestion

  it('should reply with Akkaserverless JSON for unary methods returning Any', () => {
```

**Comment by pvlugter** [comment]
Re-running build until integration tests pass (#194) ...

**Comment by pvlugter** [comment]
Released in `0.33.4`

File: sdk/test/action-handler-test.js (hunk 1 above)
Comment (johanandren) [reply sdk/test/action-handler-test.js]: Ah, ups, thanks for re-enabling

**Pull request merged**